### PR TITLE
WL-3950: Very basic availability table

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
Linked to https://github.com/ox-it/wl-reference/pull/102

**Comment by nicholaswilson100** [comment]
Linked to https://github.com/ox-it/wl-sirlouie/pull/5

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 1]
```diff
@@ -521,11 +521,10 @@ private void displayCitations(PrintWriter out, List<Citation> citations, Citatio
 			out.println("\t\t\t\t<span class=\"Z3988\" title=\""+ citation.getOpenurlParameters().substring(1).replace("&", "&amp;")+ "\"></span>");
 
 			if (citation.getSchema().getIdentifier().equals("book") || citation.getSchema().getIdentifier().equals("chapter")){
-				out.println("\t\t\t\t<div style=\"padding: 5px;\"><a href=\"" + "http://www.bbc.co.uk" + "\" target=\"_blank\">Buy It</a></div>");
 				String exportUrlAll = collection.getUrl(org.sakaiproject.citation.api.CitationService.REF_TYPE_EXPORT_RIS_SEL)
 						+ "?citationCollectionId=" + citationCollectionId + "&resourceDisplayName=" + title + "&contentCollectionId=" + contentCollectionId
 						+ "&citationId="+  citation.getId();
-				out.println("<div style=\"padding: 5px;\"><a style='' href='" +  exportUrlAll + "' id='exportList'>Export list to Ref Man</a></div></div></div>");
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java:528]: What is this todo with availability table?

Comment (nicholaswilson100) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java:528]: nothing really, I can remove it.

**Pull request merged**